cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NamespaceHttpHandler.java

```diff
@@ -162,7 +162,8 @@ public void create(HttpRequest request, HttpResponder responder, @PathParam("nam
   @Path("/unrecoverable/namespaces/{namespace-id}")
   public void delete(HttpRequest request, HttpResponder responder, @PathParam("namespace-id") String namespace) {
     if (!cConf.getBoolean(Constants.Dangerous.UNRECOVERABLE_RESET, Constants.Dangerous.DEFAULT_UNRECOVERABLE_RESET)) {
-      responder.sendStatus(HttpResponseStatus.FORBIDDEN);
+      responder.sendString(HttpResponseStatus.FORBIDDEN,
+                           "Namespace deletion is not enabled. Please enable it and restart CDAP master.");
```

Comment:
I'm wondering of this is good enough. We don't have a property "namespace deletion"; instead, we have "enable.unrecoverable.reset" . Perhaps this should be:
```
"Namespaces cannot be deleted because 'enable.unrecoverable.reset' is not enabled. Please enable it and restart CDAP Master."
```